spec/presenters/search_presenter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe SearchPresenter do
+  subject(:presenter) { SearchPresenter.new(document) }
+
+  context 'a complete document is given' do
+    let(:document) do
+      double(
+        'Document',
+        title:                    'A Title',
+        summary:                  'A summary',
+        base_path:                '/some-finder/a-title',
+        public_updated_at:        Time.now,
+        body:                     '## A Title',
+        format_specific_metadata: { country: ['GB'], blank_value: '' }
+      )
+    end
+
+    describe '#indexable_content' do
+      it 'indexes the body alone' do
+        expect(presenter.indexable_content).to eql('## A Title')
+      end
+    end
+
+    describe '#to_json' do
+      subject(:json) { presenter.to_json }
+
+      it 'has values that are present' do
+        expect(json[:title]).to eql('A Title')
+        expect(json[:link]).to eql(document.base_path)
+      end
+
+      it 'includes format-specific metadata' do
+        expect(json[:country]).to eql(['GB'])
+      end
+
+      it 'does not include blank values' do
+        expect { json.fetch(:blank_value) }.to raise_error(KeyError)
+      end
+
+      # This needs thought.
+      it 'prepares organisation details *somehow*'
```

Comment:
> "notifier rather than source of truth" seems to suggest Rummager will just go off and find stuff for itself based on, say, just content_id or base_path?

Yes. My understanding is that:

* the publishing tool will publish links to the Publishing API
* Publishing API puts a message on the message queue
* Rummager sees the message on the fanout
* Rummager fetches the links from Publishing API and indexes them